spec/amp-story-player.md

```diff
@@ -503,6 +508,18 @@ player.addEventListener('navigation', (event) => {
 })
 ```
 
+#### storyNavigation
+
+Fired when the story inside the player changes to a new page. It provides the `pageId` and `progress` of the story. The progress is the completion percentage of the story. e.g. In a story containing 5 pages, it will be 0.5 when reaching the 3rd page. And will reach 1 once the last page is reached.
```

Comment:
Actually it's a bit weird and the 3rd page would be 0.6 IIRC. Can we not give an example and say it's `....... percentage of the story between 0 and 1, [0, 1]`?

Reply:
Not the right place for this but... Why not just provide the page index and total pages? Avoid rounding issues. Every use case I can think for this event would require computing the the page index from the progress.

Reply:
Calculating progress on the web, as number of pages seen in a Story, or amount of pixels scrolled on a regular web document, is something that very often trips people. Ideally, the progress would start at 0 and end at 1, to compute the bounce rate, and completed metrics. We like to send it for this reason, otherwise you might have discrepancies between your Analytics numbers and the ones you get from the player.

What are the use cases you have in mind, where you'd need to get the page index? @Enriqe told me some people asked for a way to go back to the first page, for which we should expose a `rewindStory(storyUrl)` method as you'd want to do more than just navigate to the first page, eg: reset the analytics.

Reply:
Let's say I want to show comments/favorite-status based on what page you're on.  

Also, note when setting the page in the API you are asking to provide a delta page value, not a scroll value between 0 and 1. It's weird that I set in one unit but read the property in another unit.

Reply:
For this use case we'd recommend using page IDs, as they're more reliable than a page index.
If you really want to use the index so you don't have to maintain a map of ID <> index, you can create deterministic page IDs in your stories, eg: `<amp-story-page id="page-${index}"`, and then use `show(null, 'page-n')`

Reply:
Fair enough. But, we are talking about the `storyNavigation` event here. I want to know what page you have 'navigated' to. I am not calling `show` here. 
The user has navigated to the page with my cat's photos. I want to update my UI to show all the comments about my cat.

Reply:
Can you ping us/me on our Slack channel? https://amphtml.slack.com/
I thought you'd be able to achieve what you just mentioned using page IDs, let's chat and figure out how we can make this work

Reply:
You're right.  My brain wasn't working right. I forgot to notice the `pageId` field in the doc